node/node.go

```diff
@@ -322,6 +323,12 @@ func (n *Node) ProcessBatch(ctx context.Context, header *core.BatchHeader, blobs
 	if result.err != nil {
 		return nil, err
 	}
+	if result.keys != nil {
+		n.Metrics.AcceptBatches("stored", batchSize)
+		n.Metrics.ObserveLatency("StoreChunks", "stored", result.latency)
+		n.Logger.Debug("Store batch took", "duration:", time.Duration(result.latency*float64(time.Second)).Milliseconds())
```

Comment:
what does `time.Duration(result.latency*float64(time.Second)).Milliseconds())` do?
can we just `time.Duration(result.latency)`?

Reply:
By default it will treat the number as nanosecond, which is not desired.